packages/rpc-provider/src/substrate-connect/ScProvider.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright 2017-2022 @polkadot/rpc-provider authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import type { Config } from '@substrate/connect/dist/connector/index';
```

Comment:
```suggestion
import type { Config } from '@substrate/connect';
```

Reply:
You also have to re-export that `Config` with `export type { Config }` or something